### DISCUSSION TITLE
Exercise 5 Data acquisition?cwatm_input5min.zip

### DISCUSSION THREAD

**Discussion by behvr1** [iiasa/CWatM#45] [Q&A] [answered]
I can't seem to find the exercise 5 data under the corresponding link. Has the link been updated? Or is it a problem with my operation?
Thank you for your reply.

![image](https://user-images.githubusercontent.com/59592088/194710231-bd0935e2-d1ea-43c8-af03-cae459aadf9b.png)

Thank you for your reply.

**Comment by XiaoYu778** [comment]
Hello, I also encountered this problem, did you solve it?

**Comment by mikhailsmilovic** [comment, marked as answer]
This link is to access the FTP server.
To access it, **open up any folder on your computer and paste ftp://rcwatm:Water1090@ftp.iiasa.ac.at/ into the address bar**.
![image](https://user-images.githubusercontent.com/20219171/222756628-7eed9071-80a3-427b-bd8c-42749e91ff6d.png)
